pkgs/tools/misc/qmk/default.nix

```diff
@@ -27,13 +40,23 @@ buildPythonApplication rec {
     appdirs
     argcomplete
     colorama
-    dotty-dict
+    #dotty-dict # See https://pypi.org/project/qmk-dotty-dict/
```

Comment:
```suggestion
```

Reply:
I'd rather keep this commented so that on the next update when qmk-dotty-dict becomes dotty-dict again, it's easier to make things work